doc/adr/0011-separation-eid-implementation.md

```diff
@@ -0,0 +1,29 @@
+# 11. eID implementation in separate Microservice (Java based server due to Java SDK) and additional custom API for eService
+
+Date: 2022-07-11
+
+## Status
+
+Accepted
+
+## Context
+
+There are 2 types of online administration website (eService) available
+
+1. eService with eID integration (Magnus Flow)
+2. eService without eID integration (Paula flow)
+
+This document is a decision record for any eService who wishes to use eID with our solution (Paula Flow).
```

Comment:
Paula and Magnus Flow might not be known to everybody and maybe are subject to change in future. 
```suggestion
Some eService ("Dienstanbieter") don't have an existing eID integration in place. Therefore, we need to implement the eID integration in our backend to extend our possible customer range. 
```